internal/datanode/data_node.go

```diff
@@ -939,7 +939,11 @@ func (node *DataNode) SyncSegments(ctx context.Context, req *datapb.SyncSegments
 		numRows:      req.GetNumOfRows(),
 	}
 
-	channel.(*ChannelMeta).initPKBloomFilter(ctx, targetSeg, req.GetStatsLogs(), tsoutil.GetCurrentTime())
+	err = channel.(*ChannelMeta).initPkstats(ctx, targetSeg, req.GetStatsLogs(), tsoutil.GetCurrentTime())
```

Comment:
Maybe expose `initPkstats`? Or is there any other method to avoid type assertion here?